### PR TITLE
[Divide by 0 Error] add svd check

### PR DESCRIPTION
**Pull request by gouzil**
### PR types

Others
### PR changes

Others
### Describe

* https://github.com/PaddlePaddle/Paddle/issues/49919#issuecomment-1386647307
* https://github.com/PaddlePaddle/Paddle/issues/49927#issue-1537642462

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/test_linalg_svd_op.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#  Copyright (c) 2023 PaddlePaddle Authors. All Rights Reserved.
```
Comment (wanghuancoder) [inline python/paddle/fluid/tests/unittests/test_linalg_svd_op.py]: 单测不需要新建文件，放到python/paddle/fluid/tests/unittests/test_svd_op.py即可

File: python/paddle/tensor/linalg.py [hunk 2]
```diff
@@ -1921,6 +1921,10 @@ def svd(x, full_matrices=False, name=None):
             #                  U * UH == I
             #                  V * VH == I
     """
+
+    if x.size == 0:
+        raise ValueError("input size should not be 0")
+
```
Comment (wanghuancoder) [inline python/paddle/tensor/linalg.py]: 判断放到C++端吧，放在python会影响API性能

Comment (gouzil) [reply python/paddle/tensor/linalg.py]: 已由 https://github.com/PaddlePaddle/Paddle/pull/49951 修复

**Review by wanghuancoder — APPROVED** [review]
LGTM

**Pull request merged**